pkg/build/builder/dockerutil.go

```diff
@@ -126,6 +126,22 @@ func readInt64(filePath string) (int64, error) {
 	return val, nil
 }
 
+func readMaxStringOrInt64(filePath string) (int64, error) {
```

Comment:
Need a unit test for this function

Reply:
so per my comment in https://github.com/openshift/builder/pull/246#issue-655174072 it would make sense to add unit tests not just for this function, but some of the existing code which does not have unit tests (i.e. `readInt64`)

so you are officially asking for that @adambkaplan ?

Reply:
Yes - there is a general ask across OpenShift to improve test code coverage. Adding scoped unit tests to new functions (and in this case, new functions that extend existing functions) moves the needle in the right direction.

Reply:
unit test added @adambkaplan

Reply:
actually sit tight @adambkaplan 

per https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_builder/246/pull-ci-openshift-builder-master-unit/1397983052998643712#1:build-log.txt%3A12

may have a golang version discrepancy to sort out .... `os.WriteFile` works for me locally but is noted as undefined in CI

Reply:
sure  enough, things moved between golang 1.15 and 1.16 per the overview in https://golang.org/pkg/io/ioutil/ and openshift/builder is still at 1.15 per https://github.com/openshift/release/blob/master/ci-operator/config/openshift/builder/openshift-builder-master.yaml#L37

I know you have limited availability this afternoon @adambkaplan so I'm for now going to assume we do not want to bump golang for openshift/builder this late in the cycle, and I'll switch from os.WriteFile to io/ioutil.WriteFile

Reply:
ok unit tests now clean @adambkaplan